### PR TITLE
[BrokenLinksH2] Fixed broken link

### PR DESCRIPTION
**Pull request by kturetz**
Fixed Data Source Kind link.

### CODE REVIEW

File: powerquery-docs/HandlingResourcePath.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ As `Uri.Type` is an *ascribed type* rather than a *primitive type* in the M lang
 shared GithubSample.Contents = Value.ReplaceType(Github.Contents, type function (url as Uri.type) as any);
 ```
 
-[Data Source Kind]: https://github.com/Microsoft/DataConnectors/blob/master/docs/m-extensions.md#data-source-kind
+[Data Source Kind]: /java/api/com.azure.resourcemanager.loganalytics.models.datasourcekind?view=azure-java-preview
```
Comment (JasonWHowell) [inline powerquery-docs/HandlingResourcePath.md:68]: The provided link seems to be a completely different product, which seems incorrect. See line 18 for context. 

Looks like the previous URL was removed on purpose by the product team.
https://github.com/microsoft/DataConnectors/pull/476
In that PR, he mentioned the replacement is /power-query/startingtodevelopcustomconnectors but that page doesn't mention Data Source Kind. 


Seems hard to fix. Remove the link? I don't see other options. 

@DougKlopfenstein might need your help to weigh other options.

Comment (JasonWHowell) [reply powerquery-docs/HandlingResourcePath.md:68]: @mattmasson do we have any reference for Data Source Kind in the current docs?

**Comment by JasonWHowell** [comment]
#assign: mattmasson

**Comment by JasonWHowell** [comment]
#assign:mattmasson

File: powerquery-docs/HandlingResourcePath.md (hunk 1 above)
Comment (DougKlopfenstein) [reply powerquery-docs/HandlingResourcePath.md]: Miguel Escobar provided the correct link - /power-query/handlingdataaccess#data-source-kind

**Pull request merged**